src/req_wrap-inl.h

```diff
@@ -10,6 +10,7 @@
 namespace node {
 
 ReqWrapBase::ReqWrapBase(Environment* env) {
+  CHECK(env->has_run_bootstrapping_code());
```

Comment:
Aren't this and the HandleWrap constructor somewhat perf-sensitive?

Reply:
I don’t think this is going to make a noticeable difference, compared to all the other work that goes into setting up requests or handles